library/dash/src/main/java/com/google/android/exoplayer2/source/dash/manifest/DashManifestParser.java

```diff
@@ -901,6 +903,33 @@ protected static String parseString(XmlPullParser xpp, String name, String defau
     return value == null ? defaultValue : value;
   }
 
+  protected static int parseDolbyChannelConfiguration(XmlPullParser xpp, String name,
+      int defaultValue) {
+    String value = Util.toLowerInvariant(xpp.getAttributeValue(null, name));
+    if (value == null) {
+      return defaultValue;
+    }
+    int channels;
+    // TODO: Parse other channel configurations
+    switch (value) {
+      case "4000":
+        channels = 1;
+        break;
+      case "A000":
```

Comment:
Now you're using toLowerInvariant on L908, these case constants also need to be lowercase :)